squidb-tests/src/com/yahoo/squidb/sql/InsertTest.java

```diff
@@ -325,4 +325,26 @@ public void run() {
             }
         }, IllegalStateException.class);
     }
+
+//    public void testLowLevelInsertPerf() {
```

Comment:
We could consider that as a separate project, sure. I don't know if it'd be helpful for testing regressions automatically or not since the results would be largely relative to the hardware the tests were running on, etc., but it could be useful as our own debugging tool.